### ISSUE TITLE
InstanceNorm operator support for Vulkan devices

### ISSUE THREAD

**Issue by vjayd** [pytorch/pytorch#92285]
### 🚀 The feature, motivation and pitch

At present, only BatchNorm op is supported for Vulkan drivers. The model file throws an error when using InstanceNorm on a Vulkan device ( Adreno )

### Alternatives

I have replaced InstanceNorm with BatchNorm for the time being

**Comment by vjayd** [comment]
Hi Team, it's been more than 2 months since last post. Any updates on the same?
